google/cloud/forseti/services/server_config/service.py

```diff
@@ -26,6 +29,48 @@
 LOGGER = logger.get_logger(__name__)
 
 
+def _run(client):
+    """Runs Forseti.
+
+    Args:
+        client (ClientComposition): Forseti client object.
+
+    Returns:
+        ServerRunReply: the returned proto message.
+    """
+    today = datetime.now()
+    model_name = today.strftime('run_model_%Y%m%d_%H%M')
+    message = 'Forseti server run complete'
```

Comment:
Move this message to be closer to the `return`, where it it used. around line 71.

Reply:
Done